compiler/rustc_middle/src/ty/region.rs

```diff
@@ -336,7 +336,9 @@ pub struct EarlyParamRegion {
 
 impl std::fmt::Debug for EarlyParamRegion {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{:?}, {}, {}", self.def_id, self.index, self.name)
+        // self.def_id goes first because of `erased-regions-in-hidden-ty.rs` being impossible to write
+        // error annotations for otherwise. :)
+        write!(f, "({:?})_{}/#{}", self.def_id, self.name, self.index)
```

Comment:
```suggestion
        write!(f, "{:?}_{}/#{}", self.def_id, self.name, self.index)
```
isnÄ't it `DefId(whatever)` anyways?

Reply:
yeah ur right